packages/lesswrong/lib/collections/multiDocuments/views.ts

```diff
@@ -45,3 +52,16 @@ MultiDocuments.addView("summariesByParentId", function (terms: SummariesByParent
     },
   };
 });
+
+MultiDocuments.addView("pingbackLensPages", (terms: MultiDocumentsViewTerms) => {
+  return {
+    selector: {
+      $or: [
+        jsonArrayContainsSelector("pingbacks.Tags", terms.documentId),
+        jsonArrayContainsSelector("pingbacks.MultiDocuments", terms.documentId),
```

Comment:
Don't we also need to update the pingbacks view for tags to look inside of `pingbacks.MultiDocuments` (if the `hasWikiLenses` feature flag is enabled)?